src/sage/features/databases.py

```diff
@@ -44,16 +44,13 @@ def __init__(self):
             sage: isinstance(DatabaseConwayPolynomials(), DatabaseConwayPolynomials)
             True
         """
-        if CONWAY_POLYNOMIALS_DATA_DIR:
-            search_path = [CONWAY_POLYNOMIALS_DATA_DIR]
-        else:
-            search_path = []
-        StaticFile.__init__(self, "conway_polynomials",
-                            filename='conway_polynomials.p',
-                            search_path=search_path,
-                            spkg='conway_polynomials',
-                            description="Frank Luebeck's database of Conway polynomials",
-                            type='standard')
+        PythonModule.__init__(
```

Comment:
```suggestion
        super().__init__(
```
(not that it is that important)